### PR TITLE
Using requirements.txt inside docker

### PR DESCRIPTION
**Pull request by juusechec**
Using requirements.txt to update or install python packages.

This is related with https://github.com/opengisch/projectgenerator/pull/165

### CODE REVIEW

**Comment by juusechec** [comment]
The changes in docker-compose is a solution due problems generated in "context" of docker by having a separated `docker-compose.travis.yml` https://github.com/moby/moby/issues/34986

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-psycopg2
+psycopg2==2.7.4
```
Comment (m-kuhn) [inline requirements.txt]: Is there a problem with newer versions?

**Comment by m-kuhn** [comment]
I like :+1:

File: requirements.txt (hunk 1 above)
Comment (juusechec) [reply requirements.txt]: No, it can be psycopg2>=2.7.4

Comment (juusechec) [reply requirements.txt]: This is the latest version available, released on Feb 6.

**Pull request merged**